qiskit/primitives/base/base_sampler.py

```diff
@@ -11,9 +11,65 @@
 # that they have been altered from the originals.
 
 r"""
-===================
-Overview of Sampler
-===================
+=====================
+Overview of SamplerV2
+=====================
+
+:class:`~BaseSamplerV2` is a primitive that samples outputs of quantum circuits.
+
+Following construction, a sampler is used by calling its :meth:`~.BaseSamplerV2.run` method
+with a list of pubs (Primitive Unified Blocks). Each pub contains two values that, together,
+define a computational unit of work for the sampler to complete:
+
+* A single :class:`~qiskit.circuit.QuantumCircuit`, possibly parameterized.
+
+* A collection parameter value sets to bind the circuit against if it is parametric.
+
+Running a sampler returns a :class:`~qiskit.provider.JobV1` object, where calling
```

Comment:
This and all all references to JobV1 need to be changed to the new base type defined in #11552

Reply:
Resolving as this is a duplicate of https://github.com/Qiskit/qiskit/pull/11529#discussion_r1452385278